### DISCUSSION TITLE
有道P031 没有uresource分区 且 校验芯片失败

### DISCUSSION THREAD

**Discussion by Eganchiyu** [PenUniverse/PenMods#227] [你问我答] [answered]
![image](https://github.com/PenUniverse/PenMods-release/assets/80200957/adf11bf0-ee36-45f6-9018-8142f12a2b77)
在尝试刷p032PRO 16GB的时候，校验芯片失败，所以是不能强刷这个的是吧？
![image](https://github.com/PenUniverse/PenMods-release/assets/80200957/79e770a0-a4a8-4029-822e-1bc726e6dcff)
感谢解答
p.s.有谁的p031刷了什么成功的吗？以后适配penmods时我们应该刷到哪里？
感谢解答

**Comment by Eganchiyu** [comment]
我用adb提取出来的线刷也是没有uresources的......

**Comment by Redbeanw44602** [comment, marked as answer]
有的型号是没有uresouce的，被塞到system分区里了。
如果校验芯片失败，说明无法直接刷。

**Comment by Eganchiyu** [comment]
而且我的也是p031 CHN STD，是2.7.3，但我有同学的p031 CHN STD却是词典笔OS，整个不会了

**Reply by Eganchiyu** [reply]
三代独立的方案的话，我们要不要刷到哪里呢？（真切提问(ﾟ▽ﾟ*) ）

**Comment by YumeSemi** [comment]
![image](https://github.com/PenUniverse/PenMods-release/assets/74101017/ecf98b8c-797f-4479-b984-fc7e566cc1c2)
我的ydp031芯片校验过了，但就system刷不进去...
